doc/memcache.md

```diff
@@ -0,0 +1,39 @@
+# Memcache
+
+`freno` can be instructed to write aggregated metrics to `memcache`. This allows clients to get data directly from `memcached` without even hitting `freno` HTTP. This, in turn, decouples read load from `freno`.
+
+Instruct `freno` to write aggregated metrics to `memcache` via `MemcacheServers` in the [config file](../resources/freno.conf.sample.json). List all `memcache` servers like so:
```

Comment:
**To instruct** `freno` to write aggregated metrics to `memcache` via `MemcacheServers` in the [config file](../resources/freno.conf.sample.json), list all `memcache` servers **like this**: ?